code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -136,7 +136,7 @@
 	name = "Dragon Form"
 	desc = "Take on the shape a lesser ash drake."
 	invocation = "RAAAAAAAAWR!"
-	convert_damage = FALSE
+	convert_damage = TRUE
```

Comment:
wont this be inherited from parent? you probably dont need the line at all

Reply:
Yeah fair

Reply:
make the change pls